$NetBSD: patch-ae,v 1.6 2003/01/20 22:55:44 jschauma Exp $

If the ``Housekeeping'' lines stay, `make install; make deinstall; make install' fails.

--- Makefile.1.orig	Tue Sep 11 00:53:09 2001
+++ Makefile.1	Mon Jan 20 17:21:22 2003
@@ -88,7 +88,7 @@
 	@-$(MKDIRS) $(MAN1DIR) 2>$(DEVNULL); exit 0
 	@-test -d $(MAN5DIR) || $(RM) $(MAN5DIR); exit 0
 	@-$(MKDIRS) $(MAN5DIR) 2>$(DEVNULL); exit 0
-	@chmod 0644 $(MANS)
+	@${CHMOD} 0644 $(MANS)
 	@for a in $(MANS1S); \
   do $(INSTALL) new/$$a.1 $(MAN1DIR)/$$a.$(MAN1SUFFIX) || exit 1; \
      if test "X$(MANCOMPRESS)" != "X"; \
@@ -101,14 +101,12 @@
      then $(MANCOMPRESS) -c new/$$a.5 >$(MAN5DIR)/$$a.$(MAN5SUFFIX); \
      else :; fi; \
   done
-	echo Housekeeping file >install.man
 
 install.bin: $(BINS) $(BASENAME)
 	@-$(MKDIRS) $(BINDIR) 2>$(DEVNULL); exit 0
-	@chmod 0755 $(BINS)
+	@${CHMOD} 0755 $(BINS)
 	$(INSTALL) $(BINS) $(BINDIR)
 	@-dirname / >$(DEVNULL) || $(INSTALL) examples/dirname $(BINDIR)
-	echo Housekeeping file >install.bin
 
 install:
 	@$(MAKE) install.man install.bin
